packages/web-components/src/components/va-accordion/va-accordion.tsx

```diff
@@ -220,6 +220,7 @@ export class VaAccordion {
 
   render() {
     const { openSingle } = this;
+    const isInsideSearchFilter = this.el.classList.contains('va-search-filter__accordion')
```

Comment:
I'm handling this by looking for a class inside the va-search-filter component so that we don't expose a public property on va-accordion.